js/views/audioOnlyScreen.js

```diff
@@ -0,0 +1,83 @@
+/*********************************************************************
+  AUDIO ONLY SCREEN
+*********************************************************************/
+const React = require('React');
+const ControlBar = require('../components/controlBar');
+const ClassNames = require('classnames');
+
+const ScrubberBar = require('../components/scrubberBar');
+
+/**
+ * The screen that is shown while the player state is playing, paused, or ended for the audio only player.
+ * Will display a title with description, a control bar, and a scrubber bar.
+ */
+class AudioOnlyScreen extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      controlBarVisible: true,
+      animate: false
+    };
+  }
+
+  render() {
+    var titleStyle = {
+      color: this.props.skinConfig.startScreen.titleFont.color
+    };
+    var infoPanelClass = ClassNames({
+      'oo-state-screen-audio-title': true,
+      'oo-inactive': false,
+      'oo-flex-column': true
+    });
+    var titleClass = ClassNames({
+      'oo-state-screen-audio-info': true
+    });
+    var textStyle = {
+      'maxWidth': '70%'
+    };
+    var textClass = ClassNames({
+      'oo-text-truncate': true
+    });
+    var titleMetadata = (
+      <div style={textStyle} className={textClass}>
+        <span className={titleClass} style={titleStyle}>
+          {this.props.contentTree.title}
+        </span>
+        : {this.props.contentTree.description}
+      </div>
+    );
+    return (
+      <div className="oo-state-screen-audio oo-flex-column-parent">
+        <div className={infoPanelClass}>
+          {titleMetadata}
+        </div>
+        <div className="oo-interactive-container oo-flex-column oo-flex-column-parent">
+          <ControlBar
```

Comment:
Since the control bar displays all buttons in a row, will this approach scale if we later need to organize the elements with a different layout (i.e. multiple rows, etc.)?